src/config/lib/app.js

```diff
@@ -0,0 +1,17 @@
+module.exports.start = async () => {
+    const app = require('./express')();
+    const server = require('./sequelize');
+    require('dotenv').config();
+
+    const PORT = process.env.SERVER_PORT || 4000;
+
+    try {
+        await server.sync({force: false})
```

Comment:
@everyone Is is ok if i use this instead of seeder.js? If i use this i don't need to run seeder.js manually.

Reply:
Vai I have no idea . @UitsHabib  vai can explain it better.